server/blob/blob-export-file/src/main/java/org/apache/james/blob/export/file/SimpleBlobExportMechanism.java

```diff
@@ -0,0 +1,114 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.blob.export.file;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.james.blob.api.BlobId;
+import org.apache.james.blob.api.BlobStore;
+import org.apache.james.blob.export.api.BlobExportMechanism;
+import org.apache.james.core.MailAddress;
+import org.apache.james.core.builder.MimeMessageBuilder;
+import org.apache.james.filesystem.api.FileSystem;
+import org.apache.james.server.core.MailImpl;
+import org.apache.mailet.MailetContext;
+
+public class SimpleBlobExportMechanism implements BlobExportMechanism {
+    private static final int STRING_LENGTH = 32;
+    private static final boolean WITH_LETTERS = true;
+    private static final boolean WITH_NUMBERS = true;
+    private static final String SUBJECT = "A blob had been shared with you";
+    static final String CORRESPONDING_FILE_HEADER = "corresponding-file";
+
+    public static class Configuration {
+        private final String sharingDirectory;
+
+        public Configuration(String sharingDirectory) {
+            this.sharingDirectory = sharingDirectory;
+        }
+    }
+
+    private final MailetContext mailetContext;
+    private final BlobStore blobStore;
+    private final FileSystem fileSystem;
+    private final Configuration configuration;
+
+    SimpleBlobExportMechanism(MailetContext mailetContext, BlobStore blobStore, FileSystem fileSystem, Configuration configuration) {
```

Comment:
No `@Inject` ?

Reply:
Not needed yet. Thus I did not add them. Why anticipate?